src/test/java/testsuite/integration/container/AuroraMysqlIntegrationTest.java

```diff
@@ -607,4 +618,16 @@ public void test_PreparedStatementHashCodes() throws SQLException, IOException {
     conn.close();
   }
 
+  @RepeatedTest(100)
```

Comment:
Why do we need to run it so many times?

Reply:
When the new connection for the query timeout is made, the reader cluster endpoint doesn't always choose a reader different from the original connection so it's hard to reproduce this issue consistently.